symplyphysics/laws/optics/optical_distance_difference_from_optical_distances.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve
+
+from symplyphysics import Symbol, units, print_expression, Quantity, \
+    validate_input, validate_output
+
+# Description
+## The optical difference in the course of two rays is the difference
+## in the optical distances traversed by each of the rays
+
+# Law: delta = L2 - L1
```

Comment:
Do we really need this law? We do not have laws for force difference, energy difference, or whatever difference. What's so special about optical distance difference?

Reply:
I created this definition for possible clarification of some points in the derivation of other formulas for the difference in optical distance (For example, Young's experiment, thin wedge, Newton's rings)

This material on spatial coherence writes the difference in optical distances using this definition (both beams are in the air, so the refractive indices there are equal to 1): https://studme.org/341473/matematika_himiya_fizik/vliyanie_razmerov_istochnika_sveta_prostranstvennaya_kogerentnost